honeybee_energy/properties/extension.py

```diff
@@ -197,4 +197,36 @@ class EnergyMaterialNoMassProperties(_EnergyProperties):
 
 
 class EnergyMaterialVegetationProperties(_EnergyProperties):
-    """EnergyMaterialVegetation properties to be extended by extensions."""
\ No newline at end of file
+    """EnergyMaterialVegetation properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGlazingsProperties(_EnergyProperties):
+    """EnergyWindowMaterialGlazing properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialSimpleGlazSysProperties(_EnergyProperties):
+    """EnergyWindowMaterialSimpleGlazSys properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGasProperties(_EnergyProperties):
+    """EnergyWindowMaterialGas properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGasMixtureProperties(_EnergyProperties):
+    """EnergyWindowMaterialGasMixture properties to be extended by extensions."""
+
+
+class EnergyWindowMaterialGasCustomMixtureProperties(_EnergyProperties):
+    """EnergyWindowMaterialGasCustom properties to be extended by extensions."""
+
+
+class EnergyWindowFrameMixtureProperties(_EnergyProperties):
+    """EnergyWindowFrameMixture properties to be extended by extensions."""
```

Comment:
I'm not sure that I follow what a `FrameMixture` is. Could this just be a typo that you forgot to delete after making the `GasMixture` properties?